### PR TITLE
Changed Database access

### LINKED ISSUE #44: Investigate errors from database access
- [x] Find the source of the error messages
- [x] Remove the cause of the messages

### PR DESCRIPTION
**Pull request by NilEis**
Removed the code that automatically connects to the database at startup and added connection and disconnection code.
resolves #44

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -182,3 +182,4 @@ target/classes/logback.xml
 
 *.pem
 quotes/
+target/classes/emojis.csv
```
Comment (NilEis) [inline .gitignore:185]: I dont think this should be commited

**Review by the-codeboy — CHANGES_REQUESTED** [review]
I don't think emojis.csv should be in .gitignore

**Comment by the-codeboy** [comment]
But it is in the repo https://github.com/the-codeboy/mensabot/blob/main/src/main/resources/emojis.csv

**Comment by NilEis** [comment]
I removed emojis.csv from the gitignore file 👍

**Comment by NilEis** [comment]
And I have reverted the changes because the file was in fact not needed

**Comment by NilEis** [comment]
And I added the target folder to the gitignore

**Review by the-codeboy — APPROVED** [review]

**Pull request merged**